### PR TITLE
feat: add animation to toast

### PR DESCRIPTION
**Pull request by samobasquiat**
# Summary

A transition is added to the toast component. it changed to transform instead of opacity.

Fixes # (issue)

# How did you test this change?

You can test it by storybook. Or linking this PR to alpha branch of dapp.

# Checklist:

- [x] I have performed a self-review of my code
- [x] Implemented a user interface (UI) change, referencing our Figma design to ensure pixel-perfect precision.

### CODE REVIEW

File: widget/ui/src/components/Toast/Toast.helpers.ts [hunk 1]
```diff
@@ -5,3 +5,6 @@ export function idGenerator() {
     return id;
   };
 }
+
+export const CLOSED_DELAY = 400;
+export const OPEN_DELAY = 10;
```
Comment (RyukTheCoder) [inline widget/ui/src/components/Toast/Toast.helpers.ts:10]: Can you explain why we need "OPEN_DELAY"? I removed it and it worked properly. Also the other variable is named "CLOSED_DELAY " but it is used as transition duration so it is not just for closing the toast and it is used for opening it too. So I think you can improve the name of that variable.

**Review by RyukTheCoder — CHANGES_REQUESTED** [review]

**Review by RyukTheCoder — APPROVED** [review]

**Review by RyukTheCoder — APPROVED** [review]

File: widget/ui/src/components/Toast/Toast.helpers.ts [hunk 2]
```diff
@@ -5,3 +5,6 @@ export function idGenerator() {
     return id;
   };
 }
+
+export const TOAST_EXIT_DELAY = 400; // the delay before the toast is unmounted
+export const TOAST_TRANSITION_DURATION = 400; // the duration of the toast transition
```
Comment (RanGojo) [inline widget/ui/src/components/Toast/Toast.helpers.ts:10]: I think timing doesn't match exactly with the provided video specially in terms of exist delay. Let me double check it with the design team.

**Review by RanGojo — CHANGES_REQUESTED** [review]
Good job for the enter/exit animations. 
I think the hover animation is missed in this PR. (You could check the provided video)

File: widget/ui/src/components/Toast/Toast.helpers.ts (hunk 2 above)
Comment (samobasquiat) [reply widget/ui/src/components/Toast/Toast.helpers.ts]: I reduced the time.

**Review by RanGojo — APPROVED** [review]
@samobasquiat Nice job!

**Pull request merged**